charts/cinder-csi-plugin/templates/snapshot-controller-rbac.yaml

```diff
@@ -0,0 +1,85 @@
+{{- if .Values.csi.snapshotController.enabled -}}
+# RBAC file for the snapshot controller.
+#
+# The snapshot controller implements the control loop for CSI snapshot functionality.
+# It should be installed as part of the base Kubernetes distribution in an appropriate
+# namespace for components implementing base system functionality. For installing with
+# Vanilla Kubernetes, kube-system makes sense for the namespace.
+
+apiVersion: v1
```

Comment:
To avoid any issues, pls make sure this RBAC is compatible with version used. 2.1.3 . For example, https://github.com/kubernetes-csi/external-snapshotter/blob/v2.1.3/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml